wl/utils/Utils.java

```diff
@@ -648,60 +652,48 @@ public static long filesize(File dir) {
 	}
 
 	/**
-	 * Something has gone very, VERY wrong. Print a verbose error message to
-	 * standard output and terminate the entire server abnormally.
-	 * @param str Descriptive message where the error happened.
-	 * @param x The Exception that is responsible for this problem.
+	 * Send an e-mail.
+	 * @param email E-Mail address of the recipient.
+	 * @param subject Subject line of the mail.
+	 * @param body Message text to send.
+	 * @throws Exception If anything at all goes wrong, throw an Exception.
 	 */
-	public static void fatalError(String str, Exception x) {
-		log("#########################################################");
-		log(" VERY FATAL ERROR: " + str);
-		log("  " + x);
-		x.printStackTrace();
-		log(" Something has gone seriously wrong here.");
-		log(" Killing the server in the hope that the maintainers");
-		log(" will hurry to resolve the problems.");
-		log("#########################################################");
-		System.exit(1);
+	public static void sendEMail(String email, String subject, String body) throws Exception {
+		File message = Files.createTempFile(null, null).toFile();
+		PrintWriter write = new PrintWriter(message);
+		write.println("From: noreply@widelands.org");
+		write.println("Subject: " + subject);
+		write.println();
+		write.println(body);
+		write.print(
+		    "\n-------------------------\n"
+		    +
+		    "To change how you receive notifications, please go to https://www.widelands.org/notification/.");
```

Comment:
Should this PR wait until the website notification settings are available?

Reply:
No need IMHO, and the string issue mails already do contain this note… I did however add a switch to suppress the footer in the new admin mails as those bypass the notification settings.